app/frontend/README.md

```diff
@@ -1,45 +1,110 @@
-# Vue Skeleton Frontend
+# Common Service Get Token
 
-This is the Vue Skeleton frontend. It implements a Vue frontend with Keycloak authentication support.
+This tool will automate the process for getting setup to use common services published on [https://apistore.nrs.gov.bc.ca/store/](https://apistore.nrs.gov.bc.ca/store/).
 
-## Configuration
+## Frontend
 
-The Vue Skeleton frontend will require some configuration. The API it invokes will be locked down and require a valid JWT Token to access. We will need to configure the application to authenticate using the same Keycloak realm as the [app](../). Note that the Vue Skeleton frontend is currently designed to expect all associated resources to be relative to the original access path.
+The frontend of the application is built on the [Vue.js](https://vuejs.org/) framework as a static UI layer. The web application calls to the GETOK API (see [app](../app)) for authentication and application funtionality.
 
-## Super Quickstart
+The frontend of the application allows users to fill in the appropriate info required to create a WebADE service client and submit it to WebADE, then test token aquisition. The backend API handles communication with the WebADE REST API to create the application configuration. The backend API also creates the password required for the service client and securely returns it to the UI (see [Password Generation](https://github.com/bcgov/nr-get-token/wiki/Password-Generation) for these details)
```

Comment:
Also probably want to remove WebADE-specificity

Reply:
Changed the wording to be more ambiguous